system/lib/compiler-rt/readme.txt

```diff
@@ -14,4 +14,8 @@ Run `system/lib/update_compiler_rt.py path/to/llvm-root`
 Local Change
 ------------
 
-lib/builtins/powitf2.c: enable for wasm as well as PPC
+The differences from upstream compiler-rt are stored in our fork of llvm.  The
+changes from upstream can seen by comparing the `llvmorg-10.0.0` and
+emscripten-libs-10.0.0` branches:
```

Comment:
The actual location of the fork may be good to mention? I had to look for it for a while until I saw it was embedded in the comparison URL below.
```suggestion
The differences from upstream compiler-rt are stored in our fork of llvm (which
is at `emscripten-core/llvm-project`).

Changes from upstream can seen by comparing the `llvmorg-10.0.0` and
emscripten-libs-10.0.0` branches:
```

Reply:
OK, I actually ended up re-worded. WDYT.